README.md

```diff
@@ -619,3 +620,10 @@ See [contributing.md](CONTRIBUTING.md) for sgd contribution principles.
 ## License
 
 This project license is MIT - see the [LICENSE.md](LICENSE.md) file for details
+
+## Complimentary Plugins
```

Comment:
**typo:** compl**e**mentary instead of compl**i**mentary ?

Reply:
fixed. thanks, yikes i missed that